toc/working-groups/cf-on-k8s.md

```diff
@@ -62,5 +62,7 @@ areas:
     github: tcdowney
   repositories:
   - cloudfoundry/korifi
+  - cloudfoundry/korifi-ci
+  - cloudfoundry/korifi-secrets
   - cloudfoundry-incubator/eirini-controller
```

Comment:
As long as we're updating the charter doc, let's reflect that https://github.com/cloudfoundry-incubator/eirini-controller is now in https://github.com/cloudfoundry:

```suggestion
  - cloudfoundry/eirini-controller
  - cloudfoundry/korifi
  - cloudfoundry/korifi-ci
  - cloudfoundry/korifi-secrets
```

Reply:
done! Thanks!